lib/share_notify/push_document.rb

```diff
@@ -101,7 +101,7 @@ def delete
     # A boolean attribute would be simpler.
     def deleted?
       !otherProperties.nil? && otherProperties.any? do |p|
-        p.name == "status" # && p.property == {status: ["deleted"]}
+        p.name == "is_deleted" # && p.property == true
```

Comment:
Can you remove the code after the comment? It appears it was commented out because it wasn't needed, but then not removed.